llvm/test/Transforms/LoopUnroll/unroll-remove-redundant-dbg.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: opt < %s -S -passes=loop-unroll | FileCheck %s
```

Comment:
use llvm/utils/update_test_checks.py

Reply:
this isn't properly using update_test_checks.py, see the comments in that file. I believe the CHECK lines below need to be removed first. as the comments in update_test_checks.py say, please verify that there is a difference in IR with and without the loop unroll cleanup change

Reply:
Thanks, I use update_test_checks.py to generate CHECKs, and keep some important contents

Reply:
you shouldn't manually adjust CHECK lines generated by update_test_checks.py, just keep whatever it generates

Reply:
Thanks, keep all the  CHECK lines.